src/domain/event/EventPageContainer.tsx

```diff
@@ -65,9 +65,7 @@ const EventPageContainer: React.FC<EventPageContainerProps> = ({
   );
 
   React.useLayoutEffect(() => {
-    const isCoursePage = eventType === 'course';
-    // Only course page uses super event
-    if (superEventId && isCoursePage) {
```

Comment:
Course has some superEvent features related to dates that general event should not have! I think it should be left so, so this still needs some changes. It would be just nice to roll the super event once queried here to a subcomponent.

Reply:
Moved this if-statement to EventHero.